storage/layerupload.go

```diff
@@ -60,7 +58,6 @@ type layerUploadStore interface {
 	New(name string) (LayerUploadState, error)
 	Open(uuid string) (layerFile, error)
 	GetState(uuid string) (LayerUploadState, error)
-	SaveState(lus LayerUploadState) error
```

Comment:
Wasn't the plan to export this definition, allowing one to inject as a parameter during layer store creation? The issue with this implementation is it no longer allows supporting an alternative implementation (such as a shared store).